CHANGES.md

```diff
@@ -1,3 +1,14 @@
+## edge-18.10.4
+* CLI
+  * **New** adds the concept of service profiles to the controller
```

Comment:
Is there a user-facing impact of this change that we can describe?

Reply:
Yeah, I think we should hold off on the bullet point about service profiles until we have a user-facing feature!